src/main/java/org/fungover/thunder/ClientHandler.java

```diff
@@ -31,8 +32,8 @@ public void handleConnection(Socket clientSocket) {
                 }
             }
             removeDisconnectedClients();
-        } catch (Exception e) {
-            e.printStackTrace();
+        } catch (IOException e) {
```

Comment:
Is this needed or just for refactoring?

Reply:
Yes, Sonar doesn't like printing the stacktrace for security reasons since we then might expose what methods and classes are called. 
The right way of solving it is to replace with a log message using java.util.logger.
Add this as a field:
    private static final Logger logger = Logger.getLogger(Broker.class);
    
   And then you can log an error using logger object.